.github/workflows/ets-from-source.yml

```diff
@@ -16,15 +16,11 @@ jobs:
   test-with-edm:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest, windows-latest]
         toolkit: ['pyside6']
+        runtime: ['3.8']
+    timeout-minutes: 20
     runs-on: ${{ matrix.os }}
-    env:
```

Comment:
I don't remember why this was originally necessary, but I don't think we need it any more - similar blocks have been removed in EDM-using workflows for other ETS packages.

If this removal succeeds, I'll also update in other workflows.